src/public/js/util.js

```diff
@@ -257,7 +265,7 @@ var UTIL = (function() {
       }
     }
 
-    return remainder;
+    return trimLeadingZeros(remainder);
```

Comment:
I think semantically it'd make more sense for all of the util functions to return trimmed polynomials, then for the RS functions which use them to left-pad to size if necessary -- "separate concerns" as much as possible